build/build-image/cross/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # This file creates a standard build environment for building cross
 # platform go binary for the architecture kubernetes cares about.
 
-FROM golang:1.10.1
+FROM golang:1.10.2
```

Comment:
this still doesn't appear to exist yet.

Reply:
ah, you have that in the PR body :-), hit this trying to prepare the related PR per @ixdy's request :man_shrugging: